js/bundle.js

```diff
@@ -52,7 +55,8 @@ var overview_map = L.map('overview_map', {
     touchZoom: false,
     scrollWheelZoom: false,
     doubleClickZoom: false,
-    boxZoom: false
+    boxZoom: false,
+    minZoom: 4
```

Comment:
A general fix for "loading a large bbox attempts to load an overview map which is zoomed out too much, which also fails". This will force-set the zoom to the minimum, avoiding the issue.